### PR TITLE
Projectrequirementsdoc

### PR DESCRIPTION
**Pull request by tvp590**

### CODE REVIEW

File: Project Requirements Document.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+|       **Functional Requirements**      |                                                                                                                                                                                                              |
+|            **High Priority**           | 1. Beginners can play Tetris game with modern tetris ruleset.                                                                                                                                                |
+|                                        | 2. Beginners can learn strategies to apply into game.                                                                                                                                                            |
+|                                        | 3. Our goal is to teach number of T spins through our plateform.                                                                                                                                             |
```
Comment (Daniil-Aleshechkin-IQ) [inline Project Requirements Document.md:8]: It' s actually to teach T spins by asking the user to perform an n amount of them

File: Project Requirements Document.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+|            **Low Priority**            | 1. Add more customization to the Tetris game on our plateform for beginners. For instance, we can add teaching strategies for other Tetris blocks such as super rotation system kick table, DAS and ARP etc. |
+|                                        | 2. Add some advance learning material on higher level.                                                                                                                                                       |
+| **Technical/Performance Requirements** | 1. React.js - React.js to design attractive and interactive interface for the beginners.                                                                                                                     |
+|                                        | 2. Node.js - Node.js will be used to managed backend of the Threewide. This will help us to add more embedded game and interacting features for the users.                                                   |
```
Comment (Daniil-Aleshechkin-IQ) [inline Project Requirements Document.md:14]: Node will actually be used to render the pages of our platform. Providing the user with the embedded game and React app with the Tetris problem data

File: Project Requirements Document.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+|            **High Priority**           | 1. Beginners can play Tetris game with modern tetris ruleset.                                                                                                                                                |
+|                                        | 2. Beginners can learn strategies to apply into game.                                                                                                                                                            |
+|                                        | 3. Our goal is to teach number of T spins through our plateform.                                                                                                                                             |
+|                                        | 4. Our plateform should provide some embedded small game tasks to play.                                                                                                                                      |
```
Comment (Daniil-Aleshechkin-IQ) [inline Project Requirements Document.md:9]: 3 and 4 can be combined to one task

File: Project Requirements Document.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+|                                        | 5. We should be able to verify that the user accomplished that goal.                                                                                                                                         |
+|            **Low Priority**            | 1. Add more customization to the Tetris game on our plateform for beginners. For instance, we can add teaching strategies for other Tetris blocks such as super rotation system kick table, DAS and ARP etc. |
```
Comment (Daniil-Aleshechkin-IQ) [inline Project Requirements Document.md:11]: Super rotation system is actually part of the high priority. It's central to many T spin set ups. DAS and ARR  are also part of the modern ruleset

File: Project Roles and Responsibilities.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+|                                        |                          | 2.Ensure that the team follows all necessary Scrum practices to achieve the end goal.                                                               |
+|                                        |                          | 3. Ensure that everyone in the team is updated with the latest information regarding the project and maintain proper communication within the team. |
+|           Daniil Aleshechkin           | Lead Developer           | 1. Help scrum master to make architectural level decisions  |
+|||2. Design client-side and server-side architecture for Threewide|
+|               Tirth Patel              | Full stack web developer | 1.  Work with  team members and scrum master to design and develop Threewide platform  
+|     |                          | 2. Design client-side and server-side architecture for Threewide|
+|||3. Develop a strategy learning wiki and embedded game functionality in ThreeWide |
+|               DongYun Kim              | UI designer + vlogger      |   1. Responsible for creating attractive, user-friendly and easy-to-understand UI designs for users.|
```
Comment (Daniil-Aleshechkin-IQ) [inline Project Roles and Responsibilities.md:16]: Add manage meetings and updates with clients for DongYun

File: Project Scope Statement.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+|:---------------------------:|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|       **Project Name**      | Threewide                                                                                                                                                                                                                                         |
+|   **Project Deliverables**  | **Detailed Description**                                                                                                                                                                                                                          |
+|        Login/Register       | Threewide will provide a login/register functionality to protect user information and maintain privacy.                                                                                                                                           |
```
Comment (Daniil-Aleshechkin-IQ) [inline Project Scope Statement.md:5]: We will not provide this for either MVP 1. MVP 2 will we have logging in but we will give accounts

File: Project Scope Statement.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
+﻿| **PROJECT SCOPE STATEMENT** |                                                                                                                                                                                                                                                   |
+|:---------------------------:|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|       **Project Name**      | Threewide                                                                                                                                                                                                                                         |
```
Comment (Daniil-Aleshechkin-IQ) [inline Project Scope Statement.md:3]: Can you add separate headers for MVP 1 and MVP 2?

File: Project Requirements Document.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+|       **Functional Requirements**      |                                                                                                                                                                                                              |
+|            **High Priority**           | 1. Beginners can play Tetris game with modern tetris ruleset.                                                                                                                                                |
+|                                        | 2. Beginners can learn strategies to apply into game.                                                                                                                                                            |
+|                                        | 3. Our primary goal is to teach T spin by asking the user to perform n amount of them in small gmaes embedded on Threewide. It also includes teaching of Super Rotation System Kick Table, DAS and ARP etc.                                                                                                                                          |
```
Comment (Daniil-Aleshechkin) [inline Project Requirements Document.md:8]: Nitpick: Games is misspelled here

Comment (Daniil-Aleshechkin) [reply Project Requirements Document.md:8]: Also we won't be teaching SRS DAS and ARR. This is just what is included in the modern Tetris ruleset

File: Project Scope Statement.md [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+|   **Project Deliverables**  | **Detailed Description**                                                                                                                                                                                                                          |
+|Minimum Viable Product (MVP 1)||
+|   Strategy learning wikis   | Threewide plateform will have a strategy learning wikis which will provide required knowledge about Tetris blocks, and other techniques. The difficulty level of these strategies will start from basic and will increase as the user progresses. |
```
Comment (Daniil-Aleshechkin) [inline Project Scope Statement.md:6]: This is the other way around. MVP 2 is the wiki MVP 1 is the game with solution

File: Project Requirements Document.md [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+|            **Low Priority**            | 1. Add more customization to the Tetris game on our plateform for beginners. For instance, we can add teaching strategies for other Tetris blocks such as Z,J,L,S tetrominoes etc. |
+|                                        | 2. Add some advance learning material on higher level.                                                                                                                                                       |
+| **Technical/Performance Requirements** | 1. React.js - React.js to design attractive and interactive interface for the beginners.                                                                                                                     |
+|                                        | 2. Node.js - Node.js will be used to render the pages of our platform ThreeWide. Node.js will be used to provide an embedded game and a React application with Tetris problem data to the user.                                                 |
+|                                        |    3. MongoDB - MongoDB database will be used to store and managed essential data about users and their progress level on Threewide. |
+
+
+
```
Comment (Daniil-Aleshechkin) [inline Project Requirements Document.md:17]: Nitpick: can you remove all these new lines

File: Project Requirements Document.md (hunk 8 above)
Comment (katanallama) [reply Project Requirements Document.md]: Fixed

File: Project Requirements Document.md (hunk 10 above)
Comment (katanallama) [reply Project Requirements Document.md]: Fixed

File: Project Scope Statement.md (hunk 9 above)
Comment (katanallama) [reply Project Scope Statement.md]: Fixed

File: Activity 2/Project Requirements Document.md [hunk 11]
```diff
@@ -0,0 +1,12 @@
+﻿|   **PROJECT REQUIREMENTS DOCUMENT**    |                                                                                                                                                                                                |
```
Comment (katanallama) [inline Activity 2/Project Requirements Document.md:1]: Moved to A2 directory

File: Activity 2/Project Roles and Responsibilities.md [hunk 12]
```diff
@@ -0,0 +1,19 @@
+﻿| **PROJECT ROLES AND RESPONSIBILITIES** |                          |                                                                                                                                                    |
```
Comment (katanallama) [inline Activity 2/Project Roles and Responsibilities.md:1]: Moved to A2 directory

File: Activity 2/Project Scope Statement.md [hunk 13]
```diff
@@ -0,0 +1,11 @@
+﻿|  **PROJECT SCOPE STATEMENT**   |                                                                                                                                                                                     |
```
Comment (katanallama) [inline Activity 2/Project Scope Statement.md:1]: Moved to A2 directory

File: Activity 2/Project Scope Statement.md [hunk 14]
```diff
@@ -0,0 +1,11 @@
...
+| :----------------------------: | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|        **Project Name**        | Threewide                                                                                                                                                                           |
+|    **Project Deliverables**    | **Detailed Description**                                                                                                                                                            |
+| Minimum Viable Product (MVP 1) |                                                                                                                                                                                     |
```
Comment (katanallama) [inline Activity 2/Project Scope Statement.md:5]: resolved MPV order

File: Activity 2/Project Scope Statement.md (hunk 13 above)
Comment (katanallama) [reply Activity 2/Project Scope Statement.md:1]: edits for brevity

File: Activity 2/Project Roles and Responsibilities.md (hunk 12 above)
Comment (katanallama) [reply Activity 2/Project Roles and Responsibilities.md:1]: edits for brevity

File: Activity 2/Project Requirements Document.md (hunk 11 above)
Comment (katanallama) [reply Activity 2/Project Requirements Document.md:1]: edits for brevity

**Review by katanallama — APPROVED** [review]

**Review by dav1dk1m — APPROVED** [review]
Finished review

**Review by Daniil-Aleshechkin-IQ — APPROVED** [review]

**Review by dav1dk1m — APPROVED** [review]

**Pull request merged**